cfg/pyro.cfg

```diff
@@ -28,15 +28,15 @@ alias specialshiftoff "attack2safety; bind mouse2 +airblast"
 //--------------------------------------------------------------------------------------------
 //Viewmodel Script
 
-alias hideflames "viewmodel_fov 1; r_drawviewmodel 0"
+//alias hideflames "viewmodel_fov 1; r_drawviewmodel 0"
 
 alias pyrovmoff "alias fov hideflames; fovtest"
 alias pyrovmon "alias fov none; normalvmfov"
 
 alias fovteston "alias fovtest fov"
 alias fovtestoff "alias fovtest none"
 
-alias pyrovmtoggle pyrovmtoggle1 //Change this into "pyrovmtoggle2" if you removed hideflames down below.
+alias pyrovmtoggle pyrovmtoggle2 //Change this into "pyrovmtoggle2" if you removed hideflames down below.

```

Comment:
This is personal preference and doesn't need to be committed I think

Reply:
You are right, that is my personal preference.
I've just made a brunch with my personal pyro config.